deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -304,75 +304,146 @@ spec:
+
+    * Install the operator by following the instructions in the
+    [documentation](https://docs.openshift.com/container-platform/4.1/applications/operators/olm-adding-operators-to-cluster.html#olm-installing-operators-from-operatorhub_olm-adding-operators-to-a-cluster).
+    The KubeFed operator only needs to be installed in the host cluster for the
+    deployment, not each individual target cluster.
+    ```
+    $ kubectl get po -n <namespace>
+      NAME                               READY   STATUS    RESTARTS   AGE
+      kubefed-operator-db58d9b89-8694x   1/1     Running   0          15s
+
+    ```
+    * Create a `KubeFedWebHook` resource to instantiate an admission webhook
+    controller for KubeFed. It's recommended to install this CR with
+    `scope:Cluster` to allow the webhook controller to serve API requests coming
+    from all KubeFed control planes within the same cluster.
+    ```
+    $ cat <<-EOF | kubectl apply -n <namespace> -f -
+    ---
+    apiVersion: operator.kubefed.io/v1alpha1
+    kind: KubeFedWebHook
+    metadata:
+      name: kubefed-webhook-resource
+    spec: 
+      scope: Cluster 
+    ---
+    ```
+    * Create a `KubeFed` resource to drive the installation of a KubeFed control
+    plane setting the scope to be either `Namespaced` or `Cluster`. If you're
+    planning to federate a cluster-scoped resource type like, for example
+    `StorageClass`, create this CR with `scope:Cluster`.
+    
+    ```
+    $ cat <<-EOF | kubectl apply -n <namespace> -f -
+    ---
+    apiVersion: operator.kubefed.io/v1alpha1
+    kind: KubeFed
+    metadata:
+      name: kubefed-resource
+    spec: 
+      scope: Cluster 
+    ---
+    ```
+    ```
+    $ kubectl get po -n <namespace> 
+    NAME                                          READY   STATUS    RESTARTS   AGE
+    kubefed-admission-webhook-77f7bd89dd-92kzj    1/1     Running   0          25s
+    kubefed-controller-manager-67d5d5cc99-5t2fk   1/1     Running   0          25s
+    kubefed-controller-manager-67d5d5cc99-b865z   1/1     Running   0          25s
+    kubefed-operator-db58d9b89-8694x              1/1     Running   0          61s
+    
+    ```
+    **Important Note**: 
+    * It's recommended to deploy kubefed in either a namespace-scoped or
+    cluster-scoped mode but not both ways as deploying a cluster-scoped and
+    namespace-scoped kubefed control plane to the same cluster will result in
+    both control planes try to manage resources in the namespace of the
+    namespace-scoped control plane and the managed resources will never converge
+    on the desired state. For more information, use this
+    [link](https://github.com/openshift/federation-dev/blob/master/docs/kubefed-scope.md)
+    
 
-        curl -Ls https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc3/kubefedctl-0.1.0-rc3-linux-amd64.tgz | tar xz
+    * A KubeFed control plane does not need to be installed on each cluster that
```

Comment:
Combine this with the earlier mention of not needing to install on all clusters.